release/steps/4-generate_release_notes.mjs

```diff
@@ -8,35 +8,19 @@ await checkToken();
 
 const releasingVersion = await extractVersion();
 
-console.log(chalk.blue(`Triggering Changelog generation Pipeline`));
-
-const milestone = `APIM - ${releasingVersion}`;
+console.log(chalk.blue(`Triggering Release Notes generation Pipeline`));
 
 // Use the preconfigured payload from config folder with the good parameters
-// For changelog generation, always work on master
+// For release-notes generation, always work on master
 const body = {
-  branch: 'master',
+  branch: 'ci-changelog',
```

Comment:
```suggestion
  branch: 'master',
```

Reply:
It's for testing I'll change them at the last moment. see my 2 comments above ;)